docs/2_4_settings.md

```diff
@@ -4,35 +4,39 @@ ReVanced Manager has settings that can be configured to your liking.
 
 ## ⭐ Essential settings
 
-- ### 🔗 API URL
+- ### 🪛 Allow changing patch selections
 
-  API to use to fetch updates and ReVanced Patches from.
+  Allows the user to change the patch selections from the default selections.
 
-- ### 🧬 Sources
+- ### 🔍 Version compatibility check
 
-  Override the API and download ReVanced Patches from a different source.
+  Constrains ReVanced Patches to supported app versions. Disable this to patch any version of an app.
 
-- ### 🧪 Experimental ReVanced Patches support
+  > [!WARNING]
+  > Disabling this may cause issues if the ReVanced Patches are not compatible with the app version.  
 
-  Disable checking for the version of the app when applying ReVanced Patches.
+- ### 🧑‍🔬 Show universal patches
+
+  Reveals ReVanced Patches which can be applied to any app.
 
   > [!WARNING]  
-  > This may cause issues if the ReVanced Patches are not compatible with the app version.
+  > Because the patches generalize the app, they may not work on all apps.
 
-- ### 🧑‍🔬 Experimental universal support
+- ### 🧬 Sources
 
-  This will show or hide ReVanced Patches, which are not meant for any app in particular but apply to all apps
+  Override the API and download ReVanced Patches from a different source.
```

Comment:
`ReVanced Patches` -> `patches`

Reply:
Should `ReVanced Patches` be kept by the API URLdescription?

It currently is `API to use to fetch updates and ReVanced Patches from.`

Reply:
No, patches from sources are not necessarily ReVanced's.